scripts/potentials.py

```diff
@@ -9,73 +9,20 @@
 This is in analogy to `areas.py` but for potentials [TWh/a] rather than areas [km2] .
 """
 
-from enum import IntEnum, Enum
-
 import numpy as np
 import pandas as pd
 import rasterio
 from rasterstats import zonal_stats
 import fiona
 
-from technical_eligibility import Eligibility, FOREST, FARM, OTHER
-
-
-class ProtectedArea(IntEnum):
-    """Derived from UNEP-WCMC data set."""
-    PROTECTED = 255
-    NOT_PROTECTED = 0
-
-
-class Potential(Enum):
-    """Classes of renewable electricity potentials."""
-    ROOFTOP_PV = (1, [Eligibility.ROOFTOP_PV])
-    OPEN_FIELD_PV = (2, [Eligibility.ONSHORE_WIND_AND_PV])
-    ONSHORE_WIND = (3, [Eligibility.ONSHORE_WIND_AND_PV, Eligibility.ONSHORE_WIND])
-    OFFSHORE_WIND = (4, [Eligibility.OFFSHORE_WIND])
-
-    def __init__(self, int_id, corresponding_eligibilities):
-        self.int_id = int_id
-        self.eligible_on = corresponding_eligibilities
-
-    @property
-    def area_name(self):
-        return "{}_km2".format(self.name.lower())
-
-    @property
-    def capacity_name(self):
-        return "{}_mw".format(self.name.lower())
-
-    @property
-    def electricity_yield_name(self):
-        return "{}_twh_per_year".format(self.name.lower())
-
-    @staticmethod
-    def onshore():
-        """Returns all onshore potentials."""
-        return [
-            Potential.ROOFTOP_PV,
-            Potential.OPEN_FIELD_PV,
-            Potential.ONSHORE_WIND,
-        ]
-
-    @staticmethod
-    def offshore():
-        """Returns all offshore potentials."""
-        return [
-            Potential.OFFSHORE_WIND
-        ]
-
-    def __repr__(self):
-        return self.electricity_yield_name
-
-    def __str__(self):
-        return self.__repr__()
+from renewablepotentialslib.eligibility import Eligibility, ProtectedArea, Potential, FOREST, FARM, OTHER
 
 
 def potentials(path_to_units, path_to_eez, path_to_shared_coast,
```

Comment:
@timtroendle I modified this file to incorporate the minor differences between it and 'capacities.py', such that cross-dependencies were no longer required.